gym_miniworld/miniworld.py

```diff
@@ -550,7 +550,7 @@ def reset(
         This also randomizes many environment parameters (domain randomization)
         """
         super().reset(seed=seed)
-        self.rand = RandGen(seed)
+        self.rand, _ = seeding.np_random(seed)
```

Comment:
If we use `np_random` then there is no need for this line. `np_random` is set within the `super().reset(seed=seed)`

Reply:
This will affect the backward compatibility but will make the project compatible with the gym and gymnasium APIs. What are your thoughts?

Reply:
Given that we are removing all of the functions from `random.py` then Im not sure how much backward compatibility we should support.

Reply:
I got rid of the line you referred to and made some other changes so it passed all the tests. Is there any specific files of gym and gymnasium I can look at in order to make sure that the project matches gym and gymnasium APIs?

Reply:
If all envs pass the check_env with minimal warnings then there is no much more to do.
If you remove `self.rand, _ = seeding.np_random(seed)` then you will need to update all `rand` to `np_random`

Reply:
Sounds great! I've already made the adjustments you mentioned in the latest commit.